web/onos-gui/src/app/onos-config/networkchanges/network-changes.component.ts

```diff
@@ -109,8 +120,21 @@ export class NetworkChangesComponent extends TableBaseImpl implements OnInit {
         }
     }
 
-    rollback() {
+    rollback(selected: string): void {
+        if (selected == null || selected === 'Pending') {
```

Comment:
is it `Pending` or `pending`

Reply:
Pending for the temporary net-change, pending for the change within Configs